packages/sit-onyx/src/components/OnyxSlider/OnyxSlider.vue

```diff
@@ -0,0 +1,725 @@
+  marks: false,
+  modelValue: (props) => [props.min ?? 0] as number[],
+  orientation: "horizontal",
+  trackMode: "default",
+  tooltipDisplay: "auto",
+  disabled: FORM_INJECTED_SYMBOL,
+  showError: FORM_INJECTED_SYMBOL,
+  skeleton: SKELETON_INJECTED_SYMBOL,
+});
+
+const emit = defineEmits<{
+  /**
+   * Emitted when the validity state of the slider changes.
+   */
+  validityChange: [validity: ValidityState];
+  /**
+   * Emitted when the slider changes
+   */
+  "update:modelValue": [value: number[]];
+  /**
+   * Emitted when the slider is committed (e.g. on blur, or on pressing enter)
+   */
+  "update:committed": [value: number[]];
+}>();
+
+const modelValue = useVModel({
+  props,
+  emit,
+  key: "modelValue",
+  default: () => [props.min ?? 0] as number[],
+});
+
+const customError = computed(() => props.customError);
+const { vCustomValidity, errorMessages } = useCustomValidity({ props, emit, customError });
+const messages = computed(() => getFormMessages(props.message));
+
+defineOptions({ inheritAttrs: false });
+
+const { rootAttrs, restAttrs } = useRootAttrs();
+const { densityClass } = useDensity(props);
+
+const { disabled, showError } = useFormContext(props);
+const errorClass = useErrorClass(showError);
+const skeleton = useSkeletonContext(props);
+
+const handleChange = (values: number[]) => {
+  emit("update:modelValue", values);
+};
+
+const handleCommit = (values: number[]) => {
+  emit("update:committed", values);
+};
+
+const handleDecreaseByIcon = () => {
+  if (disabled.value) return;
+
+  if (modelValue.value.length === 1) {
+    const currentValue = modelValue.value[0] ?? props.min;
+    const stepValue = props.shiftStep ?? props.step ?? 1;
+    const newValue = Math.max(currentValue - stepValue, props.min);
+    const newValues = [newValue];
+    handleChange(newValues);
+    handleCommit(newValues);
+  }
+};
+
+const handleIncreaseByIcon = () => {
+  if (disabled.value) return;
+
+  if (modelValue.value.length === 1) {
+    const currentValue = modelValue.value[0] ?? props.min;
+    const stepValue = props.shiftStep ?? props.step ?? 1;
+    const newValue = Math.min(currentValue + stepValue, props.max);
+    const newValues = [newValue];
+    handleChange(newValues);
+    handleCommit(newValues);
+  }
+};
+
+const { min, max, step, marks, orientation } = toRefs(props);
+
+const {
+  elements: { root, rail, track, thumbContainer, thumbInput, mark, markLabel },
+  state: { focusedThumbIndex, activeThumbIndex, isDragging, marksList },
+  internals: { isMarkActive, valueToPercent, axis },
+} = createSlider({
+  values: modelValue,
+  min,
+  max,
+  step,
+  marks,
+  orientation,
+  onChange: handleChange,
+  onCommit: handleCommit,
+});
+
+const isValueControl = computed(() => props.control === "value");
+/**
+ * Icon control works only when there is a single thumb.
+ */
+const isIconControl = computed(() => props.control === "icon" && modelValue.value.length === 1);
+</script>
+
+<template>
+  <div
```

Comment:
@larsrickert Check this UX. In the current Figma design, the skeleton mode just shows a single block for simplicity.

<img width="363" height="68" alt="Image" src="https://github.com/user-attachments/assets/71b42318-efb5-415e-84fd-f8e584b67d72" />